src/api/todos.ts

```diff
@@ -0,0 +1,17 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+import { RequestTodo } from '../types/RequestTodo';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (userId: number, data: RequestTodo) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, data);
```

Comment:
USER_ID is not present in [API defenition](https://mate-academy.github.io/fe-students-api/todos.html)